Topics/Tech_Stacks/Unity_ML_Agents.md

```diff
@@ -0,0 +1,199 @@
+# Introduction to Unity's ML-Agents Toolkit
+
+Unity's ML-Agents Toolkit, or simply ML Agents, is a package that provides a simple and intuitive way to develop agents capable of interacting with both 2D and 3D environments in Unity. It is a project that aims to bring machine learning together with the many environments that can be easily developed in Unity, and provide an easy and accessible way for game developers to work with machine learning.
+
+In this introductory tutorial, we will explore the basics of reinforcement learning with an example built completely from scratch.
+
+# Installation Guide
+Before installing Unity's ML-Agents package, ensure that you installed the following:
+
+[Unity 2022.3 or Later](https://unity3d.com/get-unity/download)
+
+[Python 3.10.12 or Higher](https://www.python.org/downloads/)
+
+PyTorch, it is recommended that you create a virtual environment to avoid conflicts with any existing packages.
```

Comment:
Maybe you could add a quick tutorial on how to create a virtual environment here too

Reply:
Added a small section about virtual environments and to run everything in a terminal opened to the project directory